docs/modules/user-guide/partials/proc_deploying-a-devfile-registry-with-olm.adoc

```diff
@@ -0,0 +1,55 @@
+[id="deploying-a-devfile-registry_{context}"]
+= Deploying a devfile registry with operator lifecycle manager
+
+[role="_abstract"]
+When you have operator lifecycle manager (OLM) installed on your cluster, use the devfile registry operator to install and deploy a devfile registry.
+
+.Procedure
+..Installing the devfile registry operator
+. Install link:https://github.com/operator-framework/operator-lifecycle-manager[OLM].
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/operator-sdk/latest/[operator-sdk CLI].
+. Run `operator-sdk run bundle quay.io/devfile/registry-operator-bundle:next` to install the operator-sdk CLI.
+
+..Deploying the devfile registry
+. On an OpenShift cluster, run:
++
+====
+----
+$ cat <<EOF | oc apply -f -
+apiVersion: registry.devfile.io/v1alpha1
+kind: DevfileRegistry
+Metadata:
+  name: devfile-registry
+spec:
+  devfileIndexImage: quay.io/devfile/devfile-index:next
+EOF
+----
+====
++
+. On a Kubernetes cluster, set the k8s.ingressDomain field:
++
+====
+----
+$ cat <<EOF | kubectl apply -f -
+apiVersion: registry.devfile.io/v1alpha1
+kind: DevfileRegistry
+metadata:
+  		     name: devfile-registry
+spec:
+  devfileIndexImage: quay.io/devfile/devfile-index:next
+ 	tls:
+    enabled: false
+  k8s:
+    ingressDomain: $INGRESS_DOMAIN
+EOF
+----
+====
++
+
+Note: whether you are on the OpenShift or Kubernetes cluster, if you are deploying your own devfile registry, change the `devfileIndexImage` field to the container image that you built.
```

Comment:
Note admonitions are probably wise to use in a block construction:

```adoc
[NOTE]
====
<text here>
====
```

This is more future-proof in case you need to add more information to the note at some point later on. :)

Also, does this note only _specifically_ apply to OpenShift or Kubernetes clusters? Would it be more accurate to say "Regardless of the cluster, if you are deploying...."?

(I would change the lead-in anyway, for what it's worth: "Whether you are on an OpenShift or Kubernetes cluster".)